lib/msal-common/src/authority/Authority.ts

```diff
@@ -335,8 +338,13 @@ export class Authority {
      * Gets OAuth endpoints from the given OpenID configuration endpoint.
      */
     private async getEndpointMetadataFromNetwork(): Promise<OpenIdConfigResponse | null> {
+        const options = {};
+        if (this.proxyUrl.length) {
```

Comment:
Empty string will evaluate falsey as well
```suggestion
        if (this.proxyUrl) {
```

Reply:
TIL!